packages/next/src/cli/next-dev.ts

```diff
@@ -246,12 +259,10 @@ const nextDev: CliCommand = async (argv) => {
         try {
           // these should not error
           if (
-            configKey === 'serverComponentsExternalPackages' ||
-            configKey === 'appDir' ||
-            configKey === 'images' ||
-            configKey === 'reactStrictMode' ||
-            configKey === 'swcMinify' ||
-            configKey === 'configFileName'
+            // we only want the key after the dot for experimental options
+            supportedTurbopackNextConfigOptions
+              .map((key) => key.split('.').splice(-1)[0])
+              .includes(configKey)
```

Comment:
This check will fail even if a user provides the default values which is pretty common due to spreading the default config provided

Reply:
I didn't change the behavior here, default is handled below this condition?